angr/analyses/cfg/cfg_fast.py

```diff
@@ -1884,7 +1924,26 @@ def _scan_irsb(self, cfg_job, current_func_addr):
 
         return entries
 
+    def _create_entries_filter_target(self, target):
+        """
+
```

Comment:
Some comments on what this method does would be awesome!

Reply:
This function slipped through during the rebase. It was an old function used in cfg_fast that is not used anymore. Removed now.